tests/bootstrap.php

```diff
@@ -32,6 +32,15 @@
 	$_test_root = '/tmp/wordpress-tests-lib';
 }
 
+// Check if tests are run locally without configuring test environment.
+if ( false !== strpos( $_test_root, 'wp-phpunit/wp-phpunit' ) ) {
+	if ( ! getenv( 'WP_PHPUNIT__TESTS_CONFIG' ) || ! defined( 'WP_TESTS_CONFIG_FILE_PATH' ) ) {
+		echo 'Error: wp-tests-config.php is missing! Please use wp-tests-config-sample.php to create a config file.' . PHP_EOL;
+		echo 'Info: Alternatively, you can run `npm run wp-env start && npm run test-php` to execute tests in the wp-env environment.' . PHP_EOL;
+		exit( 1 );
+	}
+}
```

Comment:
```suggestion
if ( str_contains( $_test_root, 'wp-phpunit/wp-phpunit' ) && ! ( getenv( 'WP_PHPUNIT__TESTS_CONFIG' ) || defined( 'WP_TESTS_CONFIG_FILE_PATH' ) ) ) {
	echo 'Error: wp-tests-config.php is missing! Please use wp-tests-config-sample.php to create a config file.' . PHP_EOL;
	echo 'Info: Alternatively, you can run `npm run wp-env start && npm run test-php` to execute tests in the wp-env environment.' . PHP_EOL;
	exit( 2 );
}
```

Reply:
@westonruter `str_contains` is only PHP 8+ and not sure if polyfills will be loaded at this point of time.

Reply:
OK, right. I've updated my suggestion.